docs/contributing/README.md

```diff
@@ -118,6 +118,41 @@ Then, run the continuous batching tests:
 python -m pytest -v -x tests/e2e -m cb
 ```
 
+## Debugging
+
+!!! tip
+    You can actually `oc edit` a pod and change the image without having the pod schedule to a different node! (useful for testing if software or hardware is the issue).
+
+1. `Topology Aware Allocation`:
+      1. This mode supports users to request a special set of AIU cards based on `PCI` topology. By using this mode, we can guarantee to pick up AIU cards of a particular class in the node.  
```

Comment:
I think we should mention that this bit here is specific to the aiu operator and scheduling workloads onto specific cards (and maybe leave a comment to link out to those docs once they exist). Reading the full rendered page, it is a tiny bit confusing to have the debugging bit here jump straight into operator scheduling stuff without any explanation